platform/broadcom/sonic-platform-modules-ruijie/b6510-48vs8cq/config/x86_64_ruijie_b6510_48vs8cq_r0_config.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/python
```

Comment:
Python 2 is deprecated and no longer supported. We are in the process of migrating all Python code in SONiC to Python 3. Please change this shebang to `#!/usr/bin/env python3` and ensure the script is Python 3-compliant.